.gitignore

```diff
@@ -21,5 +21,5 @@ erl_crash.dump
 
 .vscode/
 
-# Distillery config
-/rel/
+# Ignore secrets
+*.secret.*
```

Comment:
Are there any secret files now?

Reply:
We had a cookie in a secret file but if you agree we'll let you generate a cookie as needed and decide how you want to store it.

Removed this line for the time being.

Reply:
Paul, that's a secret.

Reply:
Just kidding. There are no secret files anymore. We can remove this.